modules/agar/src/test/ts/browser/api/ClipboardTest.ts

```diff
@@ -1,103 +1,162 @@
+  const pastebinState = Singleton.value<SugarElement<HTMLElement>>();
+  const unbinderState = Singleton.value<() => void>();
+
+  const getItemData = async (item: DataTransferItem) => {
+    return new Promise((resolve) => {
+      item.getAsString((data) => {
+        resolve(data);
+      });
+    });
+  };
+
+  const assertStringItem = async (item: DataTransferItem, expected: { type: string; data: string }): Promise<void> => {
+    assert.equal(item.type, expected.type);
+    assert.equal(item.kind, 'string');
+    assert.equal(await getItemData(item), expected.data);
+  };
+
+  const assertFileItem = async (item: DataTransferItem, expected: { type: string; name: string; data: string }): Promise<void> => {
+    assert.equal(item.type, expected.type);
+    assert.equal(item.kind, 'file');
+    assert.equal(item.getAsFile().name, expected.name);
+    assert.equal(await getItemData(item), expected.data);
+  };
+
+  before(() => {
+    pastebinState.set(SugarElement.fromHtml('<div class="pastebin"></div>'));
+    pastebinState.on((pastebin) => {
+      Insert.append(SugarBody.body(), pastebin);
+
+      const cutUnbinder = DomEvent.bind(pastebin, 'cut', (evt) => {
+        const dataTransfer = evt.raw.clipboardData;
+        dataTransfer.clearData();
+        dataTransfer.setData('text/plain', 'cut-data');
+      });
+
+      const copyUnbinder = DomEvent.bind(pastebin, 'copy', (evt) => {
+        const dataTransfer = evt.raw.clipboardData;
+        dataTransfer.clearData();
+        dataTransfer.setData('text/plain', 'copy-data');
+      });
+
+      const pasteUnbinder = DomEvent.bind(pastebin, 'paste', (evt) => {
+        const dataTransfer = evt.raw.clipboardData;
+        pasteState.set(dataTransfer);
+      });
+
+      unbinderState.set(() => {
+        cutUnbinder.unbind();
+        copyUnbinder.unbind();
+        pasteUnbinder.unbind();
+      });
+    });
+  });
+
+  after(() => {
+    pastebinState.on(Remove.remove);
+    pastebinState.clear();
+    unbinderState.on((unbind) => unbind());
+    unbinderState.clear();
+  });
+
+  it('Paste text and html items', () => {
+    const pastebin = pastebinState.get().getOrDie('Could not get pastebin from state');
+
+    pasteItems(pastebin, {
+      'text/plain': 'Hello world!',
+      'text/html': '<b>Hello world!</b>'
+    });
+
+    const dataTransfer = pasteState.get().getOrDie('Could not get dataTransfer from state');
+
+    assert.equal(dataTransfer.items.length, 2);
+    assertStringItem(dataTransfer.items[0], { type: 'text/plain', data: 'Hello world!' });
+    assertStringItem(dataTransfer.items[1], { type: 'text/html', data: '<b>Hello world!</b>' });
+  });
+
+  it('Paste text and html files', () => {
+    const pastebin = pastebinState.get().getOrDie('Could not get pastebin from state');
+
+    pasteFiles(pastebin, [
+      createFileFromString('a.txt', 123, 'Hello world!', 'text/plain'),
+      createFileFromString('a.html', 123, '<b>Hello world!</b>', 'text/html')
```

Comment:
changing the text doesn't break the test for example changing in:
```
createFileFromString('a.txt', 123, 'Hello world3!', 'text/plain'),
createFileFromString('a.html', 123, '<b>Hello world4!</b>', 'text/html')
```
results in a success even if the assertion also contains the specific `data`

Reply:
changing the expectation also make it pass so I think the problem is in the checker